### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by marmeladapk**
Updated required dependencies in readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ You'll need the following dependencies:
 * libgtk-3-dev
 * libhandy-1-dev >= 0.90.0
 * liblightdm-gobject-1-dev
-* libmutter
-* libwingpanel-2.0-dev
+* libmutter-6-dev
+* libwingpanel-3.0-dev
```
Comment (danrabbit) [inline README.md:19]: I think we actually dropped the libwingpanel dep completely since we use the panel as its own process now

**Review by danrabbit — CHANGES_REQUESTED** [review]
Hey thanks for noticing this! Just one comment

File: README.md (hunk 1 above)
Comment (marmeladapk) [reply README.md]: I checked and removed wingpanel-dev package, greeter still built.

**Review by danrabbit — APPROVED** [review]
thanks!

**Pull request merged**